topics/metabolomics/tutorials/gcms/tutorial.md

```diff
@@ -0,0 +1,399 @@
+--- 
+layout : tutorial_hands_on
+enable: false
```

Comment:
It is still hidden @melpetera

Reply:
Yes, it is meant to be. We needed it to be available this way as a first step. This will allow the material to be tested by colleagues that are not familiar with the GTN contribution system. This way we can have feedback on the content, see what is missing/unclear etc. and enhance the material for it to be a ressource that can be useful for anybody. 
The difficulty for "user reviewers" to access the PR content was, I guess, one reason why this training material was left pending for so long.